source/docs/software/networktables/creating-Sendable-classes.rst

```diff
@@ -0,0 +1,195 @@
+.. _creating-Sendable-classes:
+
+
+Using the Sendable interface
+============================
+
+What is Sendable?
+-----------------
+
+Sendable is the base class that allows something to be sent
```

Comment:
Please put all text onto one line, don't limit it. Allow whatever editor you use to word-wrap it.